test_scripts/API/ButtonSubscription/005_Subscribe_button_SDL_send_UnsubscribeButton_after_Subscribe_SUCCESS_timeout.lua

```diff
@@ -0,0 +1,70 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check that SDL sends UnsubscribeButton request if HMI respond to SubscribeButton request
+--  once default timeout expires
+------------------------------------------------------------------------------------------------------------------------
+-- In case:
+-- 1) Mobile app requests SubscribeButton(<button>)
+-- 2) SDL sends Buttons.SubscribeButton(<button>, appId) to HMI
+-- 3) HMI does not respond during default timeout
+-- 4) SDL responds SubscribeButton(GENERIC_ERROR) to mobile app
+-- 5) HMI sends Buttons.SubscribeButton(SUCCESS) to SDL
+-- SDL does:
+-- 1) Send Buttons.UnsubscribeButton(<button>, appId) to HMI
+-- 2) Receive response Buttons.UnsubscribeButton(SUCCESS) and keep actual unsubscribed state for <button>
+-- 3) Not send UnsubscribeButton response to mobile app
+------------------------------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+local events = require('events')
+
+--[[ Local Variables ]]
+local appSessionId1 = 1
+local himCID
```

Comment:
himCID is used only in a function rpcGenericError. Maybe moving the variable declaration to function? (to narrow the scope)